### ISSUE TITLE
Issue running AWP when Unicode objects are present in the readme.

### ISSUE THREAD

**Issue by adwaraki** [caleb531/alfred-workflow-packager#7]
AWP crashes when the README has unicode objects because packager.py does not encode anything explicitly in lines 182 and 185.

```
Traceback (most recent call last):
  File "/Users/adwaraki/.virtualenvs/AWP/bin/awp", line 8, in <module>
    sys.exit(main())
  File "/Users/adwaraki/.virtualenvs/AWP/lib/python3.8/site-packages/awp/main.py", line 42, in main
    awp.packager.package_workflow(
  File "/Users/adwaraki/.virtualenvs/AWP/lib/python3.8/site-packages/awp/packager.py", line 236, in package_workflow
    update_workflow_readme(info, config['readme'])
  File "/Users/adwaraki/.virtualenvs/AWP/lib/python3.8/site-packages/awp/packager.py", line 185, in update_workflow_readme
    if orig_readme_hash != hashlib.sha1(info['readme']).hexdigest():
TypeError: Unicode-objects must be encoded before hashing
```

This can be corrected by changing `orig_readme_hash = hashlib.sha1(info['readme']).hexdigest()` to `orig_readme_hash = hashlib.sha1(info['readme'].encode('utf-8')).hexdigest()` by explicit specifying the encoding. 

I am not sure if this is the right way to do it, but making that change worked for me. Same fix applies to line 185.

**Comment by caleb531** [comment]
@adwaraki Thanks for reporting this! This looks like an artifact of my recent refactor to drop Python 2 support and make AWP Python 3-only.

Running some example commands in both Python 2 and Python 3 confirms this:

```bash
~ ❯ python2
Python 2.7.18 (v2.7.18:8d21aa21f2, Apr 19 2020, 20:48:48) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import hashlib
>>> hashlib.sha1('Hey ❤️')
<sha1 HASH object @ 0x7f97f817aea0>
>>> 
```

```bash
~ ❯ python3
Python 3.9.12 (main, Mar 26 2022, 15:44:31) 
[Clang 13.1.6 (clang-1316.0.21.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import hashlib
>>> hashlib.sha1('Hey ❤️')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Strings must be encoded before hashing
>>> hashlib.sha1('Hey ❤️'.encode('utf-8'))
<sha1 _hashlib.HASH object @ 0x1048a3a30>
>>> 
```

With that, I will work on this and release a bugfix version of the package soon.

**Comment by caleb531** [comment]
@adwaraki Alright, I just released v2.0.1 with the fix. Please let me know how it works on your end.
https://pypi.org/project/alfred-workflow-packager/

I'll be happy to reopen the issue if there's still a problem.

**Comment by adwaraki** [comment]
> @adwaraki Alright, I just released v2.0.1 with the fix. Please let me know how it works on your end. https://pypi.org/project/alfred-workflow-packager/
> 
> I'll be happy to reopen the issue if there's still a problem.

Everything works perfectly now. Thanks for the quick fix. :)
